app/containers/widgets/ExplorableHashContainer.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React, { Component } from 'react';
+import type { Node } from 'react';
+import { observer } from 'mobx-react';
+import { handleExternalLinkClick } from '../../utils/routing';
+
+import ExplorableHash from '../../components/widgets/hashWrappers/ExplorableHash';
+
+type Props = {
+  children: ?Node,
+  hash: string,
+  isUsed: boolean,
+};
+
+@observer
+export default class ExplorableHashContainer extends Component<Props> {
+
+  render() {
+    const seizaAddress = 'https://seiza.com/blockchain/address/';
```

Comment:
As a widget this should be in `props`.
But we can set the default value to Seiza if nothing is provided from parent.
And  `https://seiza.com/blockchain/address/` should go to some config file.